lib/beaker/logger.rb

```diff
@@ -89,6 +89,12 @@ def initialize(*args)
       dests.each {|dest| add_destination(dest)}
     end
 
+    # Setter for line_prefix that also calculates line_prefix_length
+    # @param [String] prefix The line prefix
+    def line_prefix=(prefix)
+      @line_prefix = prefix
+      @line_prefix_length = prefix.length
```

Comment:
if we're going to give up `@line_prefix_length`, we should do so entirely.  Why is it being kept/updated here?